### ISSUE TITLE
feature: Display Time 

### ISSUE THREAD

**Issue by cip123** [ChristianLightServices/TimeLight#27]
I don't know if this is possible in Gnome, but can the tray icon display the Elapsed Time? 

It would be nice to see how much I have worked today in the tray icon or how much until the scheduled time ends.

**Comment by LorenDB** [comment]
If you hover over the icon you should be able to see a tooltip with the elapsed time. This might not work in GNOME though; I'm not a GNOME user so I can't tell you. It does work on Plasma and in Windows though.

**Comment by cip123** [comment]
Yeah, unfortunately, it doesn't work on Gnome, and that's a bummer because you can't see the elapsed time otherwise.

**Comment by LorenDB** [comment]
I don't think that there's any good way to work around this issue. Could you file an issue with the team working on the GNOME system tray extension?
